examples/selection_tools/rectangle_selector_scatter.py

```diff
@@ -0,0 +1,49 @@
+"""
+Rectangle Selectors with ScatterGraphic
+=======================================
+
+Example showing how to use a `RectangleSelector` with a scatter plot.
+"""
+
+# test_example = false
```

Comment:
make this `true`, we'd want to have a ground truth screenshot for this since it's a new feature.